utils/gyb_syntax_support/AttributeNodes.py

```diff
@@ -199,22 +199,11 @@
              Child('Parameter', kind='Syntax',
                    node_choices=[
                        Child('Self', kind='SelfToken'),
-                       Child('Index', kind='DifferentiationIndexParam'),
+                       Child('Name', kind='IdentifierToken'),
                    ]),
              Child('TrailingComma', kind='CommaToken', is_optional=True),
          ]),
 
-    # differentiation-index-param -> '.' integer-literal
-    Node('DifferentiationIndexParam', kind='Syntax',
-         description='''
-         A differentiation index parameter: a period followed by an unsigned \
-         integer (e.g. `.0`)
-         ''',
-         children=[
-             Child('PrefixPeriod', kind='PrefixPeriodToken'),
-             Child('IntegerLiteral', kind='IntegerLiteralToken'),
-         ]),
-
     # differentiation-func-specifier ->
     #     ('primal' | 'adjoint' | 'jvp' | 'vjp') ':' decl-name ','?
```

Comment:
Apparently I forgot to remove primal/adjoint from this file. Can you help?

Reply:
Done in https://github.com/apple/swift/pull/22000/commits/aec0072e4eb43e7c59c6b73d980030413deaceea.